### ISSUE TITLE
Return learning rate with model object

### ISSUE THREAD

**Issue by AdrianAntico** [catboost/catboost#1455]
Enhancement request: 

Since Catboost selects a learning rate if one is not passed as an argument, it would be great to have the learning rate actually used in training returned with the model object, similar to how tree_count is returned with the model object. This way if I train a Catboost model with a lot of trees in order to find the optimal number, I can later do a final training with the optimal tree_count and the associated learning rate that led to that optimal tree_count. 

Example:
``` 
TrainModel <- catboost::catboost.train(...)
TrainModel$learning_rate
[1] 0.123456
```

**Comment by nikitxskv** [comment]
Hi @AdrianAntico !
We added this attribute.
You can rebuild R-package ([manual](https://catboost.ai/docs/installation/r-installation-github-installation.html)) and use it now or wait when we publish new CatBoost release.

**Comment by AdrianAntico** [comment]
@nikitxskv  @andrey-khropov thanks! Really appreciate it :)

**Comment by AdrianAntico** [comment]
@nikitxskv @andrey-khropov Hey guys, I reinstalled catboost and gave it a run but came up with the following error:

Error in .Call("CatBoostGetPlainParams_R", model$handle) : 
  "CatBoostGetPlainParams_R" not resolved from current namespace (catboost)

I reinstalled catboost via the code below, restarted R, and then ran the modeling code. 
```
remove.packages("catboost")
devtools::install_github('catboost/catboost', subdir = 'catboost/R-package')

# Then ran model code as before
catboost::catboost.train(learn_pool = TrainPool, test_pool = TestPool, params = base_params)
```
